src/app/resources/view-resources/resources-view.component.ts

```diff
@@ -17,7 +17,62 @@ import { Subject } from 'rxjs/Subject';
       height: 80vh;
       border: none;
     }
-  ` ]
+    .resRating {
+      display: grid;
+      max-width: 230px;
+      margin-left: auto;
+    }
+    .rating {
+      text-align: center;
+    }
+    $gray: #aaa;
+
+    body{
+      background: black;
+      color: white;
+    }
+    .container{
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      padding: 16px;
+      .section{
+        display: flex;
+        height: 200px;
+        align-items: center;
+        justify-content: center;
+        .block{
+          border: 1px solid $gray;
+          border-radius: 3px;
+          margin:16px;
+          padding: 40px;
+          .margin-vertical-8{
+            margin: 8px 0;
+          }
+        }
+      }
+    }
+
+    .star-rating{

```

Comment:
We already have styles in `styles.scss` for styling & the color of the star ratings.